contrib/Zipkin/SpanConverter.php

```diff
@@ -56,7 +54,7 @@ public function convert(Span $span)
                 'serviceName' => $this->serviceName,
             ],
             'name' => $span->getSpanName(),
-            'timestamp' => (int) ($timestamp / 1e3), // RealtimeClock in microseconds
+            'timestamp' => (int) ($start_realtime / 1e3), // RealtimeClock in microseconds
```

Comment:
I'm thinking we should probably use the monotonic clock rather than the realtime clock for this one:

https://github.com/open-telemetry/opentelemetry-php/blob/master/sdk/Trace/Clock.php#L40-L44

This is an artifact left from https://github.com/open-telemetry/opentelemetry-php/issues/73

@mattiabasone - are you interested in taking on that work alongside this PR?